handler.js

```diff
@@ -1,16 +1,54 @@
 'use strict';
 
-module.exports.hello = (event, context, callback) => {
-  const response = {
-    statusCode: 200,
-    body: JSON.stringify({
-      message: 'Go Serverless v1.0! Your function executed successfully!',
-      input: event,
-    }),
-  };
+const IncomingWebhook = require('@slack/client').IncomingWebhook;
+const AWS = require('aws-sdk');
+const s3 = new AWS.S3({'accessKeyId': process.env.aws_key_id, 'secretAccessKey': process.env.aws_secret_key});
+
+const slackHookUrl = process.env.slack_hook_url;
+const bucketName   = process.env.bucket_name;
 
-  callback(null, response);
+module.exports.handler = (event, context, callback) => {
+  let message = event.Records[0].ses.mail;
 
-  // Use this code if you don't use the http event with the LAMBDA-PROXY integration
-  // callback(null, { message: 'Go Serverless v1.0! Your function executed successfully!', event });
+  s3.getObject({
+    Bucket: bucketName,
+    Key: message.messageId
+  }, function(err, data) {
+    if (err) {
+      console.log(err, err.stack);
+      callback(err);
+    } else {
+      switch (message.commonHeaders.from[0]) {
+        case 'TraceView Alerts <alerts@tracelyticsmail.com>':
```

Comment:
Is it 100% reliable?  I'd use something like
` if (message.commonHeaders.from[0].includes('alerts@tracelyticsmail.com')) { ...`